src/services/DelegationsService.ts

```diff
@@ -172,6 +175,42 @@ class DelegationsService {
 
     return delegation.store(selectedIdentity)
   }
+
+  public static async checkTraversalStepsToParent(
```

Comment:
Is this the same as in the SDK? If yes, please just use the function from the SDK

Reply:
Removed this. I had forgotten to push my changes.